package.json

```diff
@@ -51,6 +51,7 @@
     "detect-browser": "^5.3.0",
     "dex-contracts-v2": "github:aeternity/dex-contracts-v2",
     "lodash-es": "^4.17.21",
+    "qr-code-styling": "github:sallandpioneers/qr-code-styling",
```

Comment:
In my opinion we should not add any external packages not handled by the NPM as there is a risk of someone taking over the repo.